core/src/main/java/org/jsmart/zerocode/core/kafka/client/BasicKafkaClient.java

```diff
@@ -45,10 +47,9 @@ public String execute(String brokers, String topicName, String operation, String
 
         } catch (Throwable exx) {
 
-            LOGGER.error("Exception during operation:{}, topicName:{}, error:{}", operation, topicName, exx.getMessage());
+            LOGGER.error("Exception during operation:{}, topicName:{}, error:{}", operation, topicNames.get(0), exx.getMessage());
```

Comment:
Just checking:
> topicNames.get(0)

Why only the first topic? This could potentially be any topic(s).
Can you please update to `topicNames` or `topicNames.toString()` to print all the topics